### DISCUSSION TITLE
Infra: Do not dismiss stale reviews

### DISCUSSION THREAD

**Discussion by zhztheplayer** [apache/gluten#7431] [General]
@apache/gluten-committers 

This is a proposal to remove the infra restriction for requiring review again before a PR is merged after it gets updated. After the change, a committer can decide whether to re-request review for the updated PR, or just proceed and merge. 

Related PR: https://github.com/apache/incubator-gluten/pull/7430

**Comment by FelixYBW** [comment]
sounds good.
